SPECS/snappy/snappy.spec

```diff
@@ -1,7 +1,7 @@
 Summary:        Fast compression and decompression library
 Name:           snappy
 Version:        1.1.9
-Release:        2%{?dist}
+Release:        3%{?dist}
```

Comment:
I might be a little late here, I would request though to please build `kexec-tools` & `crash` just to be sure nothing breaks in those packages as they take a dependency on `snappy`